packages/ingest/granule.js

```diff
@@ -50,45 +58,50 @@ class Discover {
   /**
    * Receives a file object and adds granule, bucket and path information
    * extracted from the collection record
-   * @param {object} file the file object
-   * @returns {object} Updated file with granuleId, bucket and path information
+   *
+   * @param {Object} file - the file object
+   * @returns {Object} - Updated file with granuleId, bucket and path information
    */
-  setGranuleInfo(_file) {
-    let granuleId;
-    const file = _file;
+  setGranuleInfo(file) {
+    const _file = file;
```

Comment:
What is this line doing?

Reply:
I think this line gets around our very irritating eslint rule: `no-param-reassign`

Reply:
Yeah, `no-param-reassign` is a regular irritation. It seems like a good practice to follow in general, but we have a lot of code that's meant to take an object, make changes, and return the changed object, so it gets annoying.